test/cxxbridge/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+corrosion_tests_add_test(cxxbridge "cxxbridge-exe")
+
+set_tests_properties("cxxbridge_run_cxxbridge-exe" PROPERTIES PASS_REGULAR_EXPRESSION [[Hello, Cxx! I am Rust!]])
```

Comment:
The `PASS_REGULAR_EXPRESSION` should match something that is actually printed. You can also leave it out for now.